api/v1alpha1/fio_types.go

```diff
@@ -57,6 +61,11 @@ type FioSpec struct {
 	// PodConfig contains the configuration for the benchmark pod, including
 	// pod labels and scheduling policies (affinity, toleration, node selector...)
 	PodConfig PodConfigurationSpec `json:"podConfig,omitempty"`
+
+	// Volume contains the configuration for the volume that the fio job should
+	// run on. If missing, no volume will attached to the job and Docker's layered
+	// fs performance will be measured
+	Volume *FioVolumeSpec `json:"volume,omitempty"`
```

Comment:
Optional flag is missing. I'm not quite sure about how helpful it is to measure docker's layered fs performance. are we sure we want to keep this?

Reply:
@wgershwitz is still valid here I think

Reply:
done